bin/reth/src/commands/db/list.rs

```diff
@@ -90,6 +90,8 @@ impl TableViewer<()> for ListTableViewer<'_> {
 
     fn view<T: Table>(&self) -> Result<(), Self::Error> {
         self.tool.provider_factory.db_ref().view(|tx| {
```

Comment:
should we just disable timeout on `DbTool` level? it's only used in CLI iiuc

Reply:
actually yeah, that makes sense

Reply:
merging this first, will make an issue